### PR TITLE
ch2 add order exercise

### PR DESCRIPTION
**Pull request by tscholl2**
added exercise about orders in quadratic extensions.

### CODE REVIEW

File: commalg.tex [hunk 1]
```diff
@@ -1163,6 +1163,14 @@ \subsection{Number fields, rings of integers, and orders}
 have finite index in $\Z[i]$.  Also the subgroup $2\Z + i\Z$ of
 $\Z[i]$ is not an order because it is not a ring.  
 
+\begin{exercise}
+	Let $K$ be a quadratic extension of $\QQ$
+	and $R$ be any order in $\O_K$.
+	Show that $\O_K/R$ is cyclic and that there is
```
Comment (williamstein) [inline commalg.tex]: cyclic --> cyclic as an abelian group

File: commalg.tex [hunk 2]
```diff
@@ -1163,6 +1163,14 @@ \subsection{Number fields, rings of integers, and orders}
...
+	a bijection between orders of $\O_K$ containing $R$
+	and divisors of $[\O_K:R]$.
```
Comment (williamstein) [inline commalg.tex]: (e.g., comes up in [citation koblitz]).

File: biblio.bib [hunk 3]
```diff
@@ -9604,3 +9605,22 @@ @book{koblitz1996p
   year={1996},
   publisher={Springer New York}
 }
+
+@article{Koblitz2011781,
+  title = "Elliptic curve cryptography: The serpentine course of a paradigm shift ",
```
Comment (williamstein) [inline biblio.bib:9610]: {T}he

File: commalg.tex [hunk 4]
```diff
@@ -1165,6 +1165,21 @@ \subsection{Number fields, rings of integers, and orders}
 have finite index in $\Z[i]$.  Also the subgroup $2\Z + i\Z$ of
 $\Z[i]$ is not an order because it is not a ring.  
 
+\begin{exercise}\label{ex:orderInQuadraticExts}
+	Let $K$ be a quadratic extension of $\QQ$
```
Comment (williamstein) [inline commalg.tex:1169]: \Q

File: commalg.tex [hunk 5]
```diff
@@ -1165,6 +1165,21 @@ \subsection{Number fields, rings of integers, and orders}
...
+	and $R$ be any order in $\O_K$.
+	Show that $\O_K/R$ is cyclic as an abelian group
+	and that there is a bijection between orders of
+	$\O_K$ containing $R$ and divisors of $[\O_K:R]$.
+\end{exercise}
+
+\begin{remark}
+	Exercise~\ref{ex:orderInQuadraticExts} is used in
+	elliptic curve cryptography to measure the number
+	of isogenies, see \cite[\S11.2]{Koblitz2011781}
```
Comment (williamstein) [inline commalg.tex:1179]: - "of isogenies. This is done in, e.g., \cite[\S11.2]{Koblitz2011781}."
- "of isogenies; this is used in \cite[\S11.2]{Koblitz2011781}". 
- "(see, e.g., \cite[\S11.2]{Koblitz2011781})"
- "; see \cite[...]..."
- ". See .." (yuck)

File: biblio.bib (hunk 3 above)
Comment (tscholl2) [inline biblio.bib:9610]: fixed in 15596b8eebc92e60bb380b6cd30c94f4c353f883

File: commalg.tex (hunk 4 above)
Comment (tscholl2) [inline commalg.tex:1169]: see 7ce1819b4a9fe46dc84152cc5ef28fb13586db67

File: commalg.tex (hunk 5 above)
Comment (tscholl2) [inline commalg.tex:1179]: fixed in b27cbb895570d3c68213619884f21aafb00e3d42

**Pull request merged**